.github/workflows/test.yml

```diff
@@ -15,11 +15,13 @@ jobs:
       uses: actions/setup-node@v3
       with:
         node-version: 16.x
-    - name: Install pnpm
-      run: npm install -g pnpm@6.35.1
     - name: Install dependencies
-      run: pnpm install
+      run: npm ci
     - name: Run unit tests
       run: npm run unit:test
     - name: Run integration tests
-      run: xvfb-run -a npm test
\ No newline at end of file
+      run: xvfb-run -a npm test
+    - name: Install vcse
```

Comment:
~vcse~ vsce

also I would write out the explicit name here ([VS Code Extension Manager](https://github.com/microsoft/vscode-vsce))

Reply:
71790e9e64af5a0b5df6ed72633cf127cb6bd2de